pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -49,6 +49,24 @@ func (pg *pgcontroller) addPod(obj interface{}) {
 	pg.queue.Add(req)
 }
 
+func (pg *pgcontroller) deletePG(obj interface{}) {
+	pod, ok := obj.(*v1.Pod)
+	if !ok {
+		klog.Errorf("Failed to convert %v to v1.Pod", obj)
+		return
+	}
+
+	pgName := helpers.GeneratePodgroupName(pod)
+	if _, err := pg.pgLister.PodGroups(pod.Namespace).Get(pgName); err != nil {
+		return
+	}
+
+	if err := pg.vcClient.SchedulingV1beta1().PodGroups(pod.Namespace).Delete(context.TODO(), pgName, metav1.DeleteOptions{}); err != nil {
```

Comment:
does it mean delete the pg once a pod deleted?

Reply:
yes